torch/csrc/distributed/rpc/tensorpipe_utils.cpp

```diff
@@ -91,11 +91,54 @@ class TensorpipeCpuConverter : public TensorpipeDeviceTypeConverter {
 
 C10_REGISTER_TENSORPIPE_DEVICE_TYPE_CONVERTER(CPU, TensorpipeCpuConverter);
 
+struct MetaBuffer {
+  void* ptr{nullptr};
```

Comment:
This field is never written to, it seems useless?

Reply:
agree, it is reverted